src/include/pbs_ifl.h

```diff
@@ -558,6 +559,18 @@ typedef struct preempt_job_info {
         char	order[PREEMPT_METHOD_HIGH + 1];
 } preempt_job_info;
 
+typedef struct svr_conn {
+	int sd;                      /* File descriptor for the open socket */
+	int secondary_sd;            /* Secondary File descriptor for the open socket */
+	int state;                   /* Connection state */
+	time_t state_change_time;    /* Connnetion state change time */
+	time_t last_used_time;       /* Last used time for the connection */
+	char svr_id[MAX_SVR_ID];     /* svr_id of the form server_name:port */
```

Comment:
remove this and MAX_SVR_ID, as its not going to used anywhere, this was needed in POC branch but not in here.
For more details see https://github.com/openpbs/openpbs/pull/2032#discussion_r490686158